frontend/src/pages/premium/waitlist.test.tsx

```diff
@@ -25,4 +63,52 @@ describe("The waitlist", () => {
       expect(results).toHaveNoViolations();
     });
   });
+
+  it("sends the user's email to Basket", async () => {
+    render(<PremiumWaitlist />);
+
+    const emailInput = screen.getByLabelText(
+      "l10n string: [waitlist-control-email-label], with vars: {}"
+    );
+    userEvent.clear(emailInput);
+    userEvent.type(emailInput, "some_email@example.com");
+
+    const submitButton = screen.getByRole("button", {
+      name: "l10n string: [waitlist-submit-label], with vars: {}",
+    });
+    userEvent.click(submitButton);
+
+    expect(global.fetch).toHaveBeenCalledTimes(1);
+    expect(global.fetch).toHaveBeenCalledWith(
+      expect.any(String),
+      expect.objectContaining({ body: expect.stringContaining("") })
```

Comment:
Heh, so what happened here is that I just wanted to match on a string in general, because the body contains form data that needs to be parsed before being able to make assertions on it (I'm doing that in the lines below this one). I couldn't find a matcher for strings, so I went for this. Then I later thought of the right way to do that, then got distracted and forgot to implement it. I now did that in e591b670.